package.json

```diff
@@ -0,0 +1,108 @@
+{
+	"name": "jupyter-keymap",
+    "publisher": "ms-toolsai",
+	"author": {
+        "name": "Microsoft Corporation"
+    },
+	"displayName": "Jupyter Keymap",
+	"description": "Jupyter keymaps for VS Code notebooks",
+    "version": "0.0.1",
+    "engines": {
+        "vscode": "^1.59.0"
+    },
+    "categories": [
+        "Notebooks"
+    ],
+    "keywords": [
+        "notebook-keymap"
+    ],
+    "contributes": {
+		"keybindings":[
+            {
+                "key": "F",
+                "when": "notebookEditorFocused && !inputFocus",
+                "command": "notebook.find"
+            },
+            {
+                "key": "K",
+                "when": "notebookEditorFocused && !inputFocus",
+                "command": "list.focusUp"
+            },
+            {
+                "key": "J",
+                "when": "notebookEditorFocused && !inputFocus",
+                "command": "list.focusDown"
+            },
+            {
+                "key": "A",
+                "when": "notebookEditorFocused && !inputFocus",
+                "command": "notebook.cell.insertCodeCellAboveAndFocusContainer"
+            },
+            {
+                "key": "B",
+                "when": "notebookEditorFocused && !inputFocus",
+                "command": "notebook.cell.insertCodeCellBelowAndFocusContainer"
+            },
+            {
+                "key": "D D",
+                "when": "notebookEditorFocused && !inputFocus",
+                "command": "notebook.cell.delete"
+            },
+            {
+                "key": "Z",
+                "when": "notebookEditorFocused && !inputFocus",
+                "command": "undo"
+            },
+            {
+                "key": "S",
+                "when": "notebookEditorFocused && !inputFocus",
+                "command": "workbench.action.files.save"
```

Comment:
@IanMatthewHuff I recall you submitting a PR to add a command that shimmed VS Code save. I didn't see any issues with just doing this, just wanted to flag in case you think this could be a problem?

Reply:
I'll confess that I'm not sure that I remember that. But I'm old 😆 . I don't see an issue with this. Doesn't seem any different from triggering any of the other commands in VS Code via the keymaps here.

Reply:
If it helps, it was this one https://github.com/microsoft/vscode-jupyter/pull/4842

Reply:
@joyceerhl Ahh, I recall that now. That was working around a specific issue with the "when" clause. Should be really easy to test if it's still an issue. Just type "s" in a notebook cell in edit nmode and see if it's trying to save at the same time as typing the 's'. Don't believe that is a problem anymore.

Reply:
Yeah I tested that, didn't seem to be an issue. Thanks for the confirmation :)